pycaret/containers/models/time_series.py

```diff
@@ -1152,32 +1153,36 @@ def _set_args(self) -> dict:
 
     @property
     def _set_tune_grid(self) -> dict:
+        # None considers both True and False and selects the best one.
         tune_grid = {
-            "use_damped_trend": [True, False],
-            "use_trend": [True, False],
             "sp": [self.sp],
+            "use_box_cox": [None],
```

Comment:
should this be included here?

Reply:
`None` tries out both True and False per their documentation and picks the best one.

Reply:
But if it's `True` it will error out, will it not?

Reply:
Yes, but not in all cases. I wanted to keep it in tune model for cases where it works, but let it error not for other cases, but just not in `create_model`. Without the True option in at least one part of the flow, the "B" part of TBATS and BATS is not functional so I don't want to disable it completely.